Stripe/STPAPIClient.m

```diff
@@ -449,3 +463,55 @@ - (void)stopPollingSourceWithId:(NSString *)identifier {
 }
 
 @end
+
+#pragma mark - Customers
+
+@implementation STPAPIClient (Customers)
+
++ (STPAPIClient *)apiClientWithEphemeralKey:(STPEphemeralKey *)key {
+    STPAPIClient *client = [[self alloc] init];
+    client.apiKey = key.secret;
```

Comment:
Should we be copying over other instance vars from ourself to this hidden sub-client (eg `client.configuration = self.configuration`)